prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -489,6 +490,7 @@ class Instance(BaseModel):
     monitoring_state: str
     instance_profile: Optional[dict]
     tags: Optional[list] = []
+    security_groups: int
```

Comment:
This is not necessary, security groups are added yet to the service.

Reply:
I think we would need to have an additional attribute stating the security groups attached to an instance, because I think it will be easier to map which security groups are attached to which instances. So I will just make that attribute 

security_groups: list - where this will store all the security group ids of a given instance